Sources/Core/JavaModelRenderer.swift

```diff
@@ -74,11 +76,36 @@ public struct JavaModelRenderer: JavaFileRenderer {
         return [props, bitmasks, [bits]]
     }
 
-    func renderModelGetters(modifiers: JavaModifier = [.public]) -> [JavaIR.Method] {
-        let getters = transitiveProperties.map { param, schemaObj in
-            JavaIR.method(modifiers, self.typeFromSchema(param, schemaObj) + " get" + param.snakeCaseToCapitalizedPropertyName() + "()") { [
+    func propertyGetterForParam(param: String, schemaObj: SchemaObjectProperty) -> JavaIR.Method {
+        // For Booleans, Integers and Doubles, make the getter method @NonNull and squash to a default value if necessary.
+        // This makes callers less susceptible to null pointer exceptions.
+        switch schemaObj.schema {
```

Comment:
can we store `param.snakeCaseToCapitalizedPropertyName() ` in a local variable before the `switch` to avoid including that in every `case` statement

Reply:
done